Userland/Libraries/LibSemVer/SemVer.cpp

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (c) 2023, Gurkirat Singh <tbhaxor@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/GenericLexer.h>
+#include <AK/StringBuilder.h>
+
+#include <LibRegex/RegexMatcher.h>
+#include <LibRegex/RegexParser.h>
+#include <LibSemVer/Semver.h>
```

Comment:
Can be one block.

Reply:
This is done by clang-format. I will fix it